front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -78,18 +83,29 @@ export class CreateF3XStep1Component implements OnInit, OnDestroy {
     private store: Store,
     private validateService: ValidateService,
     private fb: FormBuilder,
+    private f3xSummaryService: F3xSummaryService,
+    private messageService: MessageService,
+    private activatedRoute: ActivatedRoute,
     protected router: Router
   ) {}
 
   ngOnInit(): void {
-    this.store
-      .select(selectCommitteeAccount)
+    const existingReport$: Observable<F3xSummary | undefined> = this.activatedRoute.paramMap.pipe(
+      switchMap((paramMap): Observable<F3xSummary | undefined> => {
+        const id = paramMap.get('id');
+        if (id) {
+          return this.f3xSummaryService.get(parseInt(id));
+        }
+        return of(undefined);
+      })
+    );
+    combineLatest([this.store.select(selectCommitteeAccount), existingReport$])
       .pipe(takeUntil(this.destroy$))
-      .subscribe((committeeAccount) => {
+      .subscribe(([committeeAccount, existingReport]) => {
         const filingFrequency = this.userCanSetFilingFrequency ? 'Q' : committeeAccount.filing_frequency;
         this.form.addControl('filing_frequency', new FormControl());
         this.form.addControl('report_type_category', new FormControl());
```

Comment:
Why do you use addControl() here? Is it to remove the validation rules?

Reply:
these fields don't exist in the schema, but i want to keep track of them in the form

Reply:
They are added to the form when FormBuilder calls in the this.formProperties file so this call may not be necessary (maybe). We can test this out in a follow up ticket